apify-api/openapi/components/schemas/actors/UpdateActorRequest.yaml

```diff
@@ -50,3 +50,15 @@ properties:
     oneOf:
       - nullable: true
       - $ref: ./DefaultRunOptions.yaml
+  taggedBuilds:
+    type: object
```

Comment:
Can you add some description to this object so that it's obvious what the key and the value should be?